AppBuilder/platform/dataFields/ABFieldConnect.js

```diff
@@ -237,8 +237,14 @@ module.exports = class ABFieldConnect extends ABFieldConnectCore {
     *
     * @return {Promise}
     */
-   getOptions(whereClause, term, sort, editor) {
+   async getOptions(whereClause, term, sort, editor) {
       const theEditor = editor;
+
+      if (this._getOptionsToggle) clearTimeout(this._getOptionsToggle);
+      await new Promise((resolve) => {
+         this._getOptionsToggle = setTimeout(resolve, 300);
```

Comment:
Why do we set a timeout here? Why is it `300`? 

At the very least need some comments. Would it be better to use `setTimeout`  when calling the method? 
```js
setTimeout(() => this.getOptions(...args), 300);
```
Is there something we can wait for first, rather than using 300? Maybe webix `.ready()`, `.onAfterRender()` or similar?